products/statement-generator/src/components-layout/Loading.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CircularProgress } from '@material-ui/core';
+import styles from '../styles/Loading.module.css';
+
+interface LoadingProps {}
+
+const Loading: React.FC<LoadingProps> = () => {
+  return (
+    <div className={styles.wrapper}>
```

Comment:
@davidwiese can we translate this to Material UI classes also? You can make custom classes that you apply similarly to how you did here. Check out other components to see how the Material UI custom classes work.

Reply:
Hadn't done this before, good learning experience